python/json/ufoff.json

```diff
@@ -0,0 +1,38 @@
+{
+  familyName: "Oswald",
```

Comment:
I find odd to place this file here in a folde named "json" with a generic name like "ufoff.json". What is it? Maybe you can but in a folder named "examples"

Reply:
same for the ufoff.bin file